### ISSUE TITLE
Error: Cannot use the decorators and decorators-legacy plugin together

### ISSUE THREAD

**Issue by puku0x** [stylelint/postcss-css-in-js#236]
### What steps are needed to reproduce the bug?

1. Create a `.babelrc` which has `@babel/plugin-proposal-decorators`.
2. `npx stylelint "**/*.tsx"` 

### What did you expect to happen?

Errors should be reported by Stylelint.

```
src/App.tsx
 16:9   ✖  Expected "width" to come before "height"        order/properties-order                           
 20:11  ✖  Expected "display" to come before "height"      order/properties-order   
```

### What actually happened?

Nothing. I'm not joking. Nothing happens because `extract.js` ignores Babel's errors.

https://github.com/stylelint/postcss-css-in-js/blob/09af2eb5a1ba41279e099d095c268e170f02e88b/extract.js#L160-L165

Actually, it says:

```
Error: Cannot use the decorators and decorators-legacy plugin together
```

**Comment by astone123** [comment]
@puku0x could you post an example of your Babel configuration? I'm having a similar issue #247 with plugins but the error message I get isn't as helpful as this one. I think it might have something to do with my Babel configuration.

**Comment by puku0x** [comment]
@astone123 Here it is.
https://github.com/puku0x/sample-babel-react-236

It seems the recent `@babel/parser` (v7.16.5+?) causes `TypeError: p is not iterable` error.

**Comment by hudochenkov** [comment]
> - Fixed: silent fail if @babel/plugin-proposal-decorators is in user's Babel configuration ([#237](https://github.com/stylelint/postcss-css-in-js/pull/237))

**Comment by hudochenkov** [comment]
Released in `0.38.0`.
